asapdiscovery-alchemy/asapdiscovery/alchemy/schema/network.py

```diff
@@ -102,6 +104,8 @@ def _get_network_plan(self):
             return openfe.ligand_network_planning.generate_radial_network
         elif self.network_planning_method == "maximal":
             return openfe.ligand_network_planning.generate_maximal_network
+        elif self.network_planning_method == "minimal_redundant_network":
+            return openfe.ligand_network_planning.generate_minimal_redundant_network
```

Comment:
you should probably expose `mst_num` as it can be useful to increase it to >2 in some cases.

Reply:
Ah thanks I missed that was an option, I think I'll have to rework the `network_planning_method` field to be more like the `atom_mapping_engine` field so each planning method can have options as well.